src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/LineLengthCheck.java

```diff
@@ -104,6 +104,13 @@
  *   &lt;property name="fileExtensions" value="xml, properties"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>To configure check to validate {@code import} and {@code package} statements:
+ * </p>
+ * <pre>
+ * &lt;module name="LineLength"&gt;
+ *   &lt;property name="ignorePattern" value="^$"/&gt;
+ * &lt;/module&gt;
+ * </pre>
```

Comment:
Please add Java code example to config to show violations

Reply:
Done